packages/fpdart/lib/src/exit.dart

```diff
@@ -0,0 +1,71 @@
+import 'effect.dart';
+
+typedef Exit<L, R> = Either<Cause<L>, R>;
+
+sealed class Cause<L> implements Error {
```

Comment:
Oh, also, could this get its own module? I get that it's an internal detail, but `Exit` feels different than `Cause`. Maybe rename it rather than splitting it given the size of `Exit`?